send/ad_mu

```diff
@@ -477,7 +479,8 @@ print "OU added: " . $counter_add_ous . " entries.\n";
 print "OU failed: " . $counter_fail_ous . " entries.\n";
 print "Group added: " . $counter_group_add . " entries.\n";
 print "Group removed: " . $counter_group_remove . " entries.\n";
-print "Group updated: " . $counter_group_updated . " entries.\n";
+print "Group updated without errors: " . $counter_group_updated . " entries.\n";
+print "Group updated with errors: " . $counter_group_updated_with_errors . " entries.\n";
```

Comment:
You should probably add ``$counter_group_updated_with_errors`` in the condition, which exits the script with code 1 to re-schedule the propagation (few lines below).

Reply:
Added